src/storage-client/src/types/sources.rs

```diff
@@ -1725,6 +1742,21 @@ impl SourceDesc<GenericSourceConnection> {
     }
 }
 
+impl<C> SourceDesc<C> {
+    /// lala
```

Comment:
It'd be great if you could do a pass for missing docstrings before putting the PR up for review!

Reply:
I usually do but I missed this one

Reply:
Added a pre-push git hook locally